src/nevermined/Assets.ts

```diff
@@ -266,7 +329,25 @@ export class Assets extends Instantiable {
             this.logger.log('Files encrypted')
             observer.next(CreateProgressStep.FilesEncrypted)
 
+            const ddoVersion: NvmConfigVersions = {
+                id: 0,
+                updated: ddo.created,
+                checksum: ddo.checksum(ddo.shortId()),
```

Comment:
`ddo.shortId()` never changes so the checksum will always be the same. I think here checksum should be the same one as included in the `Proof` with the individual checksum of the services. e.g.:

```json
{
  "0": "0x827d62bf7f4332973d139ba284ddea757d557794149dfa407507eff7248cbb4b",
  "2": "0x90888d47746d81f1fb9f5b3ab0e959aa768b383a127051e1935e467ce52da7c4",
  "6": "0xe12598a6184f39520bb28a7d7bce51a2114d6b6a300f29346535cd3e8cbd2de8",
  "7": "0x70de5c9c818d7c3620aab0d534dee9633b04ba1c43c3a5a01925b0fdff5998a6"
}
```
- This may require the an update on the marketplace api elasticsearch mappings.
- update the spec

Reply:
checksum in versions generated via the hash of the services serialized

Reply:
we may have to rethink the use of `JSON.stringify` for crypto operations. I don't think its deterministic